carbonmark-api/src/routes/projects/get.ts

```diff
@@ -52,280 +54,93 @@ const schema = {
-            }
-            if (
-              parseFloat(pooledProjectsData.carbonOffsets[index].balanceNCT) >=
-              1
-            ) {
-              const isDefault =
-                POOL_INFO.nct.defaultProjectTokenAddress.toLowerCase() ===
-                project.projectAddress.toLowerCase();
-              const priceKey = isDefault
-                ? "defaultPrice"
-                : "selectiveRedeemPrice";
-              uniqueValues.push(poolPrices.nct[priceKey]);
-            }
-            if (
-              parseFloat(pooledProjectsData.carbonOffsets[index].balanceBCT) >=
-              1
-            ) {
-              const isDefault =
-                POOL_INFO.bct.defaultProjectTokenAddress.toLowerCase() ===
-                project.projectAddress.toLowerCase();
-              const priceKey = isDefault
-                ? "defaultPrice"
-                : "selectiveRedeemPrice";
-              uniqueValues.push(poolPrices.bct[priceKey]);
-            }
-          });
-        }
-      }
-
-      let price = 0;
-      if (project.listings?.length) {
-        project.listings.forEach((item) => {
-          if (
-            !/^0+$/.test(item.leftToSell) &&
-            item.active != false &&
-            item.deleted != true
-          ) {
-            uniqueValues.push(item.singleUnitPrice);
-          }
-        });
-
-        const lowestPrice = uniqueValues.length
-          ? uniqueValues.reduce((a, b) =>
-              a.length < b.length ? a : a.length === b.length && a < b ? a : b
-            )
-          : "0";
-        price = lowestPrice;
-
-        /** We only want to hide duplicate projects that we have found a price for */
-        if (Number(price) !== 0 || isNil(price)) {
-          /** I THINK we are setting this to false so that there are not duplicate projects displayed */
-          pooledProjectsData.carbonOffsets[index].display = false;
-        }
-      }
-      const cmsData = findProjectWithRegistryIdAndRegistry(
+    // ----- Carbonmark Listings ----- //
+    const projects = projectData.projects.map((project) => {
+      const cmsProject = findProjectWithRegistryIdAndRegistry(
         projectsCmsData,
         project.projectID,
         project.registry
       );
-      project.description = cmsData ? cmsData.description : undefined;
-      project.name = cmsData ? cmsData.name : project.name;
-      project.methodologies = cmsData ? cmsData.methodologies : [];
 
-      project.short_description = cmsData?.projectContent
-        ? cmsData.projectContent.shortDescription
-        : undefined;
-      project.long_description = cmsData?.projectContent
-        ? cmsData.projectContent.longDescription
-        : undefined;
+      // Find the lowest price
+      // @todo change to number[]
+      const listingPrices = compact(project.listings)
+        ?.filter(isListingActive)
+        .map(extract("singleUnitPrice"));
+      const lowestPrice = listingPrices.sort().at(0);
```

Comment:
maybe we can make use of lodash's min here? it's a little clearer

https://lodash.com/docs/#min

Reply:
Good idea, done